src/Twilio/TwiML/TwiML.cs

```diff
@@ -134,6 +143,16 @@ public string ToString(SaveOptions formattingOptions = SaveOptions.None)
         }
     }
 
+    /// <summary>
+    /// Class for GenericNode object
+    /// </summary>
+    public class GenericNode : TwiML
```

Comment:
I think you could delete this class and have `AddChild` return a `new TwiML`. (You would need to remove the `abstract` modifier from `TwiML`.)